### DISCUSSION TITLE
我們要建DREAM十萬人電報社區 https://t.me/dream_BSC

### DISCUSSION THREAD

**Discussion by sean-go-ai** [Benefitspace/dream#11] [General]
一个事业、一个目标、一个信念；创新传播规律；我有一个梦想。人们追随信念，是在追随这个同样属于他们自己的信念——“我有一个梦想”。
DreamCoin是让每一个人获得梦想，实现梦想，我们需要将这个梦想用最友好完整和无私的方式呈现出来，让它成为每一个志愿者和社区用户的梦想。
不用太多，让我们每个人都拥有10亿的Dream。
币安智能链（BSC）DreamCoin 合约
0x3bfE9F4e25ed2793610BD78aFe766871F62AA11c

我們要建DREAM十萬人電報社區 https://t.me/dream_BSC

**Comment by westlife29BTC** [comment]
![Uploading image.png…]()

**Reply by sean-go-ai** [reply]
現在還沒有VI，避免使用用網上搜來的圖片喔

**Reply by junmo-moon** [reply]
梦想者，你不是还有60%的空投吗，建议你分批次的发放，可以找一些微博的大V合作，类似于财经克币姐这种一次性上万人的空投，这样可以把每个人手上的额度分散开，增加币的持仓地址，促进我们的梦想快速前行，如果单个的空投数额太大，容易引起不必要的砸盘。加油！
